project-base/tests/ShopBundle/Acceptance/acceptance/CartCest.php

```diff
@@ -105,14 +115,19 @@ public function testAddToCartFromProductDetail(
         FloatingWindowPage $floatingWindowPage
     ) {
         $me->wantTo('add product to cart from product detail');
-        $me->amOnPage('/22-sencor-sle-22f46dm4-hello-kitty/');
```

Comment:
hmm do we have some test that test friendlyURL generation for domain?

Reply:
yes, tehre is `FriendlyUrlFactoryTest`